README.md

```diff
@@ -87,35 +87,137 @@ Usage as a library
 
 The Synapse client can be used to write software that interacts with the Sage Bionetworks Synapse repository.
 
-### Example
+### Examples
 
-    import synapseclient
+#### Log-in and create a Synapse object
+```
+import synapseclient
 
-    syn = synapseclient.Synapse()
+syn = synapseclient.Synapse()
+
+## log in using auth token
+syn.login(authToken='auth_token')
+```
+
+#### Create a manifest TSV and sync a directory to synapse
```

Comment:
Nit: I think it's great this is in the readme, but eventually, we should have an article with this sort of information.

Reply:
I'm think of dropping the 'many files' part in lieu of "more than one"

Reply:
Got it.  Looking at your benchmarking, i did notice that there are times where `sync` is not as optimal of a method, but we can still just recommend this approach.

Heres an issue we should resolve: https://sagebionetworks.jira.com/browse/SYNPY-1179.

Reply:
> Nit: I think it's great this is in the readme, but eventually, we should have an article with this sort of information.

I agree - I am going to make a ticket at the top of the backlog to absorb this kind of information into https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/docs/getting_started/basics.rst